.infrastructure/98_variables.tf

```
@@ -24,4 +24,19 @@ variable "tags" {
 ### required for security group ALB, ECS and RDS
 variable "cms_app_port" {
   default = 1337
+}
+
+variable "fargate_cpu" {
+  description = "Fargate instance CPU units to provision (1 vCPU = 1024 CPU units)"
+  default     = "4096" ##### 4 vCPU
```

Let's start with lower values

```suggestion
  default     = "1024"
```

---

ok, I'll make the change with the next commit